data/org.freedesktop.impl.portal.Settings.xml

```diff
@@ -43,6 +43,29 @@
           Unknown values should be treated as 0 (no preference).
         </para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>org.freedesktop.appearance button-placement (asas)</term>
+        <listitem><para>
+          Indicates the system's preferred arrangement of buttons on the titlebar.
+          The first string array in the tuple is the left side of the window,
+          and the second string array is the right side of the window.
```

Comment:
"Left" and "right" are going to be "right" and "left" in right-to-left language locales.

We use "leading"/"start" and "trailing"/"end" for these cases; otherwise you'll have to expand that to something like this:

```suggestion
          The first string array in the tuple is the left side of the window,
          and the second string array is the right side of the window. In RTL
          locales, the first string array defines buttons in the right side of
          the window, and the second array defines buttons in the left side
          of the window.
```

Reply:
I don't think I like this, then you'd get inconsistent behavior across languages. Is there no better solution? How does this even end up getting reversed in RTL locales? Can it not be described without a visual representation? 

The first string array should always be on the left, regardless of locale, and vice versa. It shouldn't depend on the language.

Reply:
That's not how window decorations work: RTL locales flip the order of the buttons on **every** platform.